.devcontainer/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM node:latest as nodejs
+FROM mcr.microsoft.com/devcontainers/base:ubuntu
+
+ENV DOCKER_USER_ID 501 
+ENV DOCKER_USER_GID 20
+ENV BOOT2DOCKER_ID 1001
+ENV BOOT2DOCKER_GID 50
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV TZ=Europe/Berlin
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+
+ENV REFRESHED_AT 2022-12-20
+ENV CONFIGURATION_DIR_PATH supporting_files/configuration
+ENV SCRIPTS_DIR_PATH supporting_files/scripts
+
+
+# Tweaks to give Apache/PHP write permissions to the app
+RUN usermod -u ${BOOT2DOCKER_ID} www-data && \
+    usermod -G staff www-data && \
+    groupmod -g $(($BOOT2DOCKER_GID + 10000)) $(getent group $BOOT2DOCKER_GID | cut -d: -f1) && \
+    groupmod -g ${BOOT2DOCKER_GID} staff
+
+# Install packages
+# Suppress an apt-key warning about standard out not being a terminal. Use in this script is safe.
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get -y install software-properties-common cron
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com --recv 4F4EA0AAE5267A6C
+# install apt deps (first row for puppeteer)
+RUN apt-get -y install libnss3 libnspr4 libatk1.0-0 libatk-bridge2.0-0 libcups2 libdrm2 libxkbcommon0 libxcomposite1 libxdamage1 libxfixes3 libxrandr2 libgbm1 libasound2 postfix python3-setuptools wget jq sudo git
+# Cleanup
+RUN apt-get -y autoremove && \
+    apt-get -y clean
+
+# Add image configuration and scripts
+ADD ${SCRIPTS_DIR_PATH}/post-create.sh /post-create.sh
+RUN chmod 755 /*.sh
+
+# Time Zone
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+######################### DEVELOPMENT ENVIRONMENT ADDITIONS ###############################
+# install latest git & zsh
+RUN add-apt-repository ppa:git-core/ppa \
+    && apt-get update \
+    && apt-get -y install --no-install-recommends git \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && update-locale
+
+# specifics to user vscode
+ARG USERNAME="vscode"
+ARG USERHOME="/home/${USERNAME}"
+RUN usermod --shell /usr/bin/zsh ${USERNAME} \
+    && usermod -a -G www-data ${USERNAME} \
+    && usermod -a -G staff ${USERNAME}
+
+# Nodejs Installation
```

Comment:
I am confused at this point:
Comment 1: Nodejs Installation
Comment 2: NVM and NodeJS Installation

is nodejs getting installed twice? if so, wouldn't the steps related to comment 1 then be superfluous?


/UPDATE

After taking a look at post-create.sh, I think Comment 2 is just misleading, right?

Reply:
Yeah, my bad I forgot to edit the comment, I have made some changes.